mirdata/datasets/da_tacos.py

```diff
@@ -0,0 +1,640 @@
+
+{
+	"hpcp": numpy.ndarray,
+	"label": numpy.str_,
+	"track_id": numpy.str_
+}
+
+
+
+"""
+from typing import Tuple, Optional
+
+from jams import JAMS
+import numpy as np
+import json
+import shutil
+
+import librosa
+import os
+
+from mirdata import download_utils
+from mirdata import jams_utils
+from mirdata import core
+import logging
+import deepdish as dd
+
+LICENSE_INFO = """
+Creative Commons Attribution Non Commercial Share Alike 4.0 International
+"""
+BIBTEX = """@inproceedings{yesiler2019,
+    author = "Furkan Yesiler and Chris Tralie and Albin Correya and Diego F. Silva and Philip Tovstogan and Emilia G{\'{o}}mez and Xavier Serra",
+    title = "{Da-TACOS}: A Dataset for Cover Song Identification and Understanding",
+    booktitle = "Proc. of the 20th Int. Soc. for Music Information Retrieval Conf. (ISMIR)",
+    year = "2019",
+    pages = "327--334",
+    address = "Delft, The Netherlands"
+}"""
+REMOTES = {
+    "metadata": download_utils.RemoteFileMetadata(
+        filename="da-tacos_metadata.zip",
+        url="https://zenodo.org/record/3520368/files/da-tacos_metadata.zip?download=1",
+        checksum="b8aed83c45687a6bac76de3da1799237",
+        destination_dir=".",
+    ),
+    "benchmark_cens": download_utils.RemoteFileMetadata(
+        filename="da-tacos_benchmark_subset_cens.zip",
+        url="https://zenodo.org/record/3520368/files/da-tacos_benchmark_subset_cens.zip?download=1",
+        checksum="842a8112d7ece43059d3f04dd4a3ee65",
+        destination_dir=".",
+    ),
+    "benchmark_crema": download_utils.RemoteFileMetadata(
+        filename="da-tacos_benchmark_subset_crema.zip",
+        url="https://zenodo.org/record/3520368/files/da-tacos_benchmark_subset_crema.zip?download=1",
+        checksum="c702a3b97a60081311bf8e7fae7b433b",
+        destination_dir=".",
+    ),
+    "benchmark_hpcp": download_utils.RemoteFileMetadata(
+        filename="da-tacos_benchmark_subset_hpcp.zip",
+        url="https://zenodo.org/record/3520368/files/da-tacos_benchmark_subset_hpcp.zip?download=1",
+        checksum="f92cf3d00cc3195572381d6bbcc086de",
+        destination_dir=".",
+    ),
+    "benchmark_key": download_utils.RemoteFileMetadata(
+        filename="da-tacos_benchmark_subset_key.zip",
+        url="https://zenodo.org/record/3520368/files/da-tacos_benchmark_subset_key.zip?download=1",
+        checksum="f4e6b05fa9ab46002357f371a8b0e97e",
+        destination_dir=".",
+    ),
+    "benchmark_madmom": download_utils.RemoteFileMetadata(
+        filename="da-tacos_benchmark_subset_madmom.zip",
+        url="https://zenodo.org/record/3520368/files/da-tacos_benchmark_subset_madmom.zip?download=1",
+        checksum="8beb1d8fa39f95b79d5f502a41fd5f0c",
+        destination_dir=".",
+    ),
```

Comment:
I don't see the MFCC for the benchmark subset here.

Reply:
you are right!

Reply:
I removed it in one commit. When I remove the files with all the features together :S

Reply:
done 
6e0a62d